lambda/MAS/Views/ContentView.cshtml

```diff
@@ -39,6 +39,9 @@
     <link href="/app.css" rel="stylesheet" type="text/css" />
 </head>
 <body>
+    <!--*|IF:MC_PREVIEW_TEXT|*--> 
+    <!--[if !gte mso 9]><!----><span class="mcnPreviewText" style="display:none; font-size:0px; line-height:0px; max-height:0px; max-width:0px; opacity:0; overflow:hidden; visibility:hidden; mso-hide:all;">*|MC_PREVIEW_TEXT|*</span><!--<![endif]--> 
```

Comment:
Can we use either a class or inline styles? I'd prefer a class, as mailchimp will inline the styles for us automatically